lib/src/image_inlining/image_dart_compiling.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'image_inliner.dart';
+import 'package:path/path.dart';
+
+/// Data holder for image-related information: its file name and contents.
+class ImageInformation {
+  final String fileName;
+  final List<int> contents;
```

Comment:
I'd be more specific: ImageData { fileName, bytes } (as it's otherwise not obvious if that's really full integers, etc)